<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsTv</class>
 <widget class="QWidget" name="SettingsTv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>385</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabChannels">
      <attribute name="title">
       <string>Channels</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelDefaultList">
         <property name="text">
          <string>Default channel list:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelHd">
         <property name="text">
          <string>Default HD category:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editHd">
         <property name="minimumSize">
          <size>
           <width>225</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelRadio">
         <property name="text">
          <string>Default radio category:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editRadio">
         <property name="minimumSize">
          <size>
           <width>225</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="checkUdpxy">
         <property name="text">
          <string>Enable UDPXY support</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelUdpxyUrl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Server address:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="BrowseWidget" name="channels" native="true"/>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="layoutUdpxy">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLineEdit" name="editUdpxyUrl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>225</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="editUdpxyPort">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>1234</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSchedule">
      <attribute name="title">
       <string>Schedule</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="labelType">
         <property name="text">
          <string>Source type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioFile">
         <property name="text">
          <string>Local file</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="radioNetwork">
         <property name="text">
          <string>Download file</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelSource">
         <property name="text">
          <string>Source:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="BrowseWidget" name="browseXmltv" native="true">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelOffset">
         <property name="text">
          <string>Time offset:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinOffset">
         <property name="suffix">
          <string> h</string>
         </property>
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowseWidget</class>
   <extends>QWidget</extends>
   <header>common/widgets/BrowseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>editUdpxyUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>editUdpxyPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelUdpxyUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
